CHANGELOG.md

```
@@ -1016,6 +1038,13 @@ All notable changes to Zync are documented in this file. The format is based on
 [2.18.0]: https://github.com/zync-sh/zync/compare/v2.17.0...v2.18.0
 [#92]: https://github.com/zync-sh/zync/issues/92
 [#93]: https://github.com/zync-sh/zync/issues/93
+[#94]: https://github.com/zync-sh/zync/pull/94
```

markdownlint-cli2 (0.23.2)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "`#94`"

(MD053, link-image-reference-definitions)